### PR TITLE
Update Creating Projects

### PR DESCRIPTION
**Pull request by some2**
The click paths are now correct, the "COMPLETED : Project that has been completed." is guessed.

### CODE REVIEW

File: _includes/project-new.md [hunk 1]
```diff
@@ -51,17 +51,18 @@ Record the short name in your worksheet. <img src="pics/blue_pencil.png">
...
+  * CANCELLED : Project was scrapped because we did not get funding/samples/REB
     approval.
-  * Proposed : Project Initiation form has been received and is awaiting go-ahead
+  * PROPOSED : Project Initiation form has been received and is awaiting go-ahead
     by Genomics leadership.
-  * Pending : We are waiting for the project to start.
-  * Approved : Project has been approved by Genomics leadership.
+  * PENDING : We are waiting for the project to start.
+  * APPROVED : Project has been approved by Genomics leadership.
+  * COMPLETED : Project that has been completed.
```
Comment (callunity) [inline _includes/project-new.md]: :+1:

File: _includes/project-new.md [hunk 2]
```diff
@@ -96,7 +98,7 @@ sequencing type, select `Other`.
 
 ## 2.2 My Projects tab
 
-1. Click again on the _My Projects_ tab.
+1. Click _My Projects_ in the _Tracking_ section.
```
Comment (callunity) [inline _includes/project-new.md]: I'm not sure this is accurate, as _My Projects_ is a tab. I don't believe there's a _My Projects_ link in the _Tracking_ section, but are you seeing something different?

File: _includes/project-new.md [hunk 3]
```diff
@@ -83,8 +84,9 @@ specific to this project and used by MISO to track the project internally.
 
 In order to make sure MISO is SRA-compliant, you must add a _Study_ to your Project. Studies are legacy objects that are not really used in the current system, but need to be added anyway.
 
-1. Click on the _Studies_ section to expand the section.
-1. Click _Add_ at the top left of the table.
+1. Click the _My Projects_ tab at the top of the MISO dashboard.
+1. Click on your Project _Name_ to expand it.
+1. Click _Add_ at the top left of the _Studies_ table.
```
Comment (callunity) [inline _includes/project-new.md]: The expansion happens on the Studies section of the project page, not on the project page itself. What about:
```
1. Click the _My Projects_ tab at the top of the MISO dashboard.
1. Click on your Project _Name_.
1. If the _Studies_ table is not initially visible, click on the _Studies_ section to expand it.
1. Click _Add_ at the top left of the _Studies_ table.
```

File: _includes/project-new.md [hunk 4]
```diff
@@ -51,17 +51,18 @@ Record the short name in your worksheet. <img src="pics/blue_pencil.png">
 {% endif %}
 1. In the _Description_ field, enter `MISO training workshop [Date]`
 1. Choose the amount of _Progress_ that your project has accomplished:
-  * Unknown : I was told to enter this data and I does what they tells me.
-  * Active : We are receiving/have received samples and are actively working on
+  * UNKNOWN : I was told to enter this data and I does what they tells me.
+  * ACTIVE : We are receiving/have received samples and are actively working on
```
Comment (apmasell) [inline _includes/project-new.md]: Can you mention that only active and pending projects will be processed by analysis?

**Comment by callunity** [comment]
Closing as these changes have been added by other pull requests. Thank you for pointing these out!

**Pull request closed without merging**